ONSdigital/ssdc-rm-ddl
# Originating user for new case events are the person who clicked process

# Motivation and Context

Support Tool user who clicked "Process" sample button should be originating user of newCase events
# What has changed



New columns added and job entity updated.
# How to test?



See the [support tool PR](https://github.com/ONSdigital/ssdc-rm-support-tool/pull/101)
# Links



https://trello.com/c/DMFLX8em/2878-support-tool-user-who-clicked-process-sample-button-should-be-originating-user-of-newcase-events-8
# Screenshots (if appropriate):
